src/Tribe/Aggregator/Tabs/New.php

```diff
@@ -281,13 +281,15 @@ public function get_result_messages( $record, $result ) {
 			}
 
 			if ( $result && ! $messages ) {
-				__( 'No events were imported or updated.', 'the-events-calendar' );
+				$messages['error'][] = __( 'No events were imported or updated.', 'the-events-calendar' );
```

Comment:
I wouldn't consider this an error condition. You can add this to `$messages['success']`